circle2/do-eks-deployment.sh

```diff
@@ -34,11 +34,13 @@ echo "=== Updating Image Tags ==="
 if [ ! -z "${KUBERNETES_APPLICATIONS}" ]; then
   IFS=',' app_array=($KUBERNETES_APPLICATIONS)
   for application in "${app_array[@]}"; do
-    sed "s~^  tag: .*~  tag: ${DOCKER_IMAGE_TAG}~g" -i "src/environments/${ENVIRONMENT}/${application}.yaml"
-    git add "src/environments/${ENVIRONMENT}/${application}-values.yaml"
+    [[ $application == *-service ]] || microservice_name="${application}-service"
```

Comment:
This will be removed once we update the `project.env` file for existing microservices on NGI.